### ISSUE TITLE
[BUG] Website is down since weeks ago

### ISSUE THREAD

**Issue by Panquesito7** [Humans-of-Julia/HoJ-website#13]

**Comment by Azzaare** [comment]
Fair enough. It is likely due to the configuration of Google Domains on which I don't have control. Maybe someone else in @Humans-of-Julia/admins know better

**Comment by uncomfyhalomacro** [comment]
i dont have any control of the website either.

**Comment by arubhardwaj** [comment]
@Azzaare I have the control on this, let me check what happened? Is it becasue of google domains only? Are we sure?

**Comment by Azzaare** [comment]
I will checkout the repo and see if it works locally
